src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
```

Comment:
This styles are redundant, all elements in your card are blocks, so they started from the new line in column by default